unit-tests/native/src/test/scala/org/scalanative/testsuite/posixlib/PsuedoTerminalTest.scala

```diff
@@ -0,0 +1,175 @@
+package org.scalanative.testsuite.posixlib
```

Comment:
Well aware this is draft.

Reporting things as I find them.

Does  
Perhaps rename this file to Termios_PseuoTerminalTest.scala or such. That is 
clearly associate it with POSIX .h  it is testing.